docset/windows/addsadministration/Get-ADReplicationSite.md

```diff
@@ -53,7 +53,7 @@ This command gets all Active Directory Replication sites.
 
 ### Example 2: Get replication sites by flag
 ```
-PS C:\> Get-ADReplicationSite -Properties * -Filter {WindowsServer2003KCCSiteLinkBridgingEnabled -eq $TRUE}
+PS C:\> Get-ADReplicationSite -Properties * -Filter "WindowsServer2003KCCSiteLinkBridgingEnabled -eq `$TRUE"
```

Comment:
This line seems to be missing a backtick.

Reply:
Thank you!